src/main/java/org/jenkinsci/test/acceptance/docker/fixtures/SshAgentContainer.java

```diff
@@ -40,4 +42,11 @@ public String getEncryptedEd25519PrivateKeyPassphrase() {
         return resource("ed25519.pass").asText();
     }
 
+    public static SshSlaveLauncher configureSSHSlaveLauncher(DumbSlave agent, String host, int port) {
```

Comment:
```suggestion
    public SshSlaveLauncher configureSSHSlaveLauncher(DumbSlave agent) {
```
this should not be static and obtain the host and port from the current instance, callers will need to be updated.